sections/futures/Trade/ManagePosition.tsx

```diff
@@ -8,15 +8,14 @@ import Loader from 'components/Loader';
 import { useFuturesContext } from 'contexts/FuturesContext';
 import { previewErrorI18n } from 'queries/futures/constants';
 import { setOpenModal } from 'state/app/reducer';
-import { selectOpenModal } from 'state/app/selectors';
```

Comment:
Comment is for this component. We display the `orderError` here, should we still display transaction errors even though they would display in the notifier:
![image](https://user-images.githubusercontent.com/10401554/213315219-5e94e0fa-8e0f-4f2c-8d64-595cc0a069d6.png)

Not related to changes you made, but an improvement to displaying the unformatted errors.

Reply:
Yea makes sense, I've removed the transaction error for now but eventually when we do some more clean up around errors we probably want to push all these order / preview errors through the notifier.